libs/angular-routing/schematics/ng-add/schema.ts

```diff
@@ -0,0 +1,7 @@
+export interface RouterOptions {
+  project: string;
+  module?: string;
+  name: string;
```

Comment:
I think `name` can be removed, as it isn't added to `schema.json`
```suggestion
```

Reply:
> I think `name` can be removed, as it isn't added to `schema.json`

It is used internally by findModuleFromOptions